### ISSUE TITLE
Social-media placeholder images are going to break because we've renamed the default images

### ISSUE THREAD

**Issue by stevejalim** [mdn/developer-portal#1490]
So we need to add them back into the repo

**Comment by stevejalim** [comment]
Example of the OpenGraph markup

```

    <meta property="og:site_name" content="Mozilla Developer">
  





  
  
    
    <meta property="og:image" content="https://developer.mozilla.com/static/img/placeholders/article.jpg?h=0324221c33f4">
  
  <meta property="og:image:width" content="960">
  <meta property="og:image:height" content="720">


<meta property="og:url" content="https://developer.mozilla.com/">
<meta name="twitter:card" content="summary_large_image">
```

So we need to ensure we have a with the following name in the repo, still, and in the original dimensions:

- [ ] `article.jpg`

**Comment by stevejalim** [comment]
So we just duplicate `post_3_2.jpg` as `article.jpg`

**Comment by stevejalim** [comment]
Resolved in 2478fc24
